### PR TITLE
Add relative suboptimality, suboptimality and optimality plots

### PR DESCRIPTION
**Pull request by ngazagna**
instead of `convergence_curve`, I created 3 types of plots without modifying the saved df
- `suboptimality_curve` : $F(x) - F(x*)$, the default one
- `relative_suboptimality_curve` : $(F(x) - F(x*)) / (F(x^0) - F(x*))$
- `optimality_curve` : $F(x)$

### CODE REVIEW

File: benchopt/viz/plot_suboptimality_curve.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        The benchmark results.
+    benchmark : str
+        The path to the benchmark folder.
+
+    Returns
+    -------
+    fig : instance of matplotlib.figure.Figure
+        The matplotlib figure.
+    """
+    plot_suboptimality_curve(df, benchmark, relative=True)
```
Comment (agramfort) [inline benchopt/viz/plot_suboptimality_curve.py:82]:
```suggestion
    return plot_suboptimality_curve(df, benchmark, relative=True)
```

**Comment by agramfort** [comment]
@ngazagna I pushed here so we can see the images it produces on circleci

**Comment by agramfort** [comment]
see https://397-227720375-gh.circle-artifacts.com/0/dev/auto_examples/index.html
https://397-227720375-gh.circle-artifacts.com/0/dev/auto_examples/plot_run_benchmark.html#sphx-glr-auto-examples-plot-run-benchmark-py
https://397-227720375-gh.circle-artifacts.com/0/dev/auto_examples/plot_run_benchmark_python_R_julia.html#sphx-glr-auto-examples-plot-run-benchmark-python-r-julia-py

does it look good to you?

**Comment by josephsalmon** [comment]
image

"Did not converge" seems to long.
and also:
what about changing `use_acceleration` to `acc` everywhere (would avoid too long strings)?

**Comment by josephsalmon** [comment]
so:
`acceleration`
instead of
`use_acceleration=True`?

**Comment by josephsalmon** [comment]
my bad, `accelerated`instead

**Comment by agramfort** [comment]
@ngazagna you have a conflict to fix

**Comment by ngazagna** [comment]
ready to merge I think

**Comment by agramfort** [comment]
thanks @ngazagna 

let's now rename viz to plotting and get back to the stochastic log reg benchmarks.

let's keep the ball rolling !

**Pull request merged**